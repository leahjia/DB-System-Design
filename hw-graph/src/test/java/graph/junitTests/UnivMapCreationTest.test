# University Map Creation testing file.
# Tests that a graph can be created, a single node can be created and removed

# Create a graph
CreateGraph UnivMap

# Add a pair of nodes
AddNode UnivMap A
AddNode UnivMap B

# Remove a pair of nodes
RemoveNode UnivMap A
RemoveNode UnivMap B
