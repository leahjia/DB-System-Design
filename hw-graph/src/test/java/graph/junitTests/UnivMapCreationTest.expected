# University Map Creation testing file.
# Tests that a graph can be created, a single node can be created and removed

# Create a graph
created graph UnivMap

# Add a pair of nodes
added node A to UnivMap
added node B to UnivMap

# Remove a pair of nodes
removed node A from UnivMap
removed node B from UnivMap
